server/pulp/server/webservices/views/utils.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+
+def generate_django_response(response, content=None, default=None, content_type='application/json'):
```

Comment:
I would put "json" in the name, since serializing to json is the main point of having this. Maybe ``generate_json_response``.